gcc-13.yaml

```diff
@@ -28,6 +28,7 @@ environment:
       - make
       - mpc-dev
       - mpfr-dev
+      - openssf-compiler-options
       - patch
       - texinfo
```

Comment:
Is this hunk intentional? Why no epoch bump?

Reply:
This is intentional.  I initially had more extensive gcc-13 changes but merged it into here to build successfully. I'll bump it.

Reply:
Actually I'll remove it and fix it in the gcc pr I already have.